packages/opentelemetry-context-zone-peer-dep/src/ZoneContextManager.ts

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+/// <reference types="zone.js" />
```

Comment:
Does this result in potential issues with end users requiring to install zone.js to avoid missing types even they don't use zone.js?

Reply:
Since the types from `zone.js` are not exported by `ZoneContextManager` (they are used in private fields and methods, which are type-erased in the output dts files),  this directive is omitted in the output d.ts. So I believe it would not affect end users.